.github/workflows/build.yml

```diff
@@ -53,8 +53,20 @@ jobs:
     services:
       registry:
         image: registry:2
-        ports: 
+        ports:
           - 5000:5000
+      postgres:
+        image: postgres
```

Comment:
Worth specifying a version?

Reply:
Yeah good shout I think we use v11 I'll get it updated